decompile/General/231/231_033_RB_Warpball_TurnAround.c

```diff
@@ -35,11 +35,11 @@ void DECOMP_RB_Warpball_TurnAround(struct Thread* t)
     inst->matrix.t[2] += ((int)tw->vel[2] * gGT->elapsedTimeMS) >> 5;
     
 	// increment counter
-	tw->turnAround++;
-    
+	sVar3 = tw->turnAround++;
+
 	if (
 			// if count too high
-			(0x78 < sVar3) || 
+			(0x78 < sVar3) || //svar3 = tw->turnAround?
```

Comment:
tried to replace sVar3 in the if statement with simply `tw->turnAround`, but this *somehow*???? causes incorrect behavior